README.md

```diff
@@ -16,8 +28,8 @@ The HyP3 object interactions with the HyP3 API are done using an instance of the
 ```python
 >>> from hyp3_sdk import HyP3
 >>> hyp3 = HyP3(username='MyUsername', password='MyPassword')  
->>> job = hyp3.submit_rtc_job(name='MyNewJob', granule='S1A_IW_SLC__1SSV_20150621T120220_20150621T120232_006471_008934_72D8')
->>> job = hyp3.wait(job)
+>>> job = hyp3.submit_rtc_job(granule='S1A_IW_SLC__1SSV_20150621T120220_20150621T120232_006471_008934_72D8', name='MyNewJob')
```

Comment:
It might be worth breaking this up to:

```python
>>> from hyp3_sdk import HyP3
>>> hyp3 = HyP3(username='MyUsername', password='MyPassword') 

>>> granule = 'S1A_IW_SLC__1SSV_20150621T120220_20150621T120232_006471_008934_72D8'
>>> job = hyp3.submit_rtc_job(granule=granule, name='MyNewJob')
>>> job = hyp3.watch(job)
>>> job.download_files()
```
So that we don't have to have a left-right scroll bar:
![image](https://user-images.githubusercontent.com/7882693/99846013-0f451480-2b22-11eb-8262-78711843300a.png)